### DISCUSSION TITLE
PP Leaderboards for ranked maps

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#29999] [Ideas] [answered]
The current ranking system is based on score, which is a system that is outdated and has been replaced by the pp system with ppv1 coming out in 2012. However, this only happened for players rankings. Now it's time for beatmaps to follow suit. I don't think that I have to explain the flaws within the score system, however I shall do so just in case. 

Since the score system is based on combo, a play that is significantly better than the other can still be ranked lower due to a lower combo. Also, let's make two plays up. Both plays A and B have only 300s besides one miss. Now both misses are on circles of equal difficulty, however A missed in the middle of the map and B has the miss at the end. Obviously both plays are equally good but due to the combos being different, A is weighted way higher than B.
Another problem is the inability for the scoring system to understand the difficulty of an object. Here, all objects are worth the same amount of score, even though they have a completely different difficuly attached to them. 

Now all the problems mentioned here and more which the scoring system has are solved by the pp system, which is why it replaced the score system in 2012 as the offical way to weight players' plays and their overall performance.
Therefore I think that ranked beatmaps should have their leaderboards based on pp rather then score.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13024
https://github.com/ppy/osu/discussions/28423
